spec/queries/types/user_projects_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'All user projects', type: :request do
+  let(:user) { create(:user) }
+  let(:project) { create(:project, user: user) }
+  let(:task) { create(:task, project: project) }
+  let(:comment) { create(:comment, task: task) }
+  let(:headers) { user.create_new_auth_token }
+  let(:query) do
+    <<-GRAPHQL
+        query {
+          user(id: "#{user.id}") {
```

Comment:
I think that you can remove id param for user as you can fetch user from `context[:current_user]`
also I didn't see any authorization, does it mean that each user can fetch projects of other users? If yes, then please change logic to fetch user from `context[:current_user]`